app/javascript/controllers/sub_nav_controller.js

```diff
@@ -24,7 +24,7 @@ export default class extends Controller {
     e.preventDefault()
     document.querySelector('.selected')?.classList.remove('selected')
     e.target.classList.add('selected')
-    this.displayedIdValue = e.target.getAttribute('displayedId')
+    this.displayedIdValue = e.target.getAttribute('data-displayedId')
```

Comment:
Could you use [dataset](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset) here?